etc/prepare-tests.sh

```diff
@@ -4,12 +4,6 @@ cd "$(dirname "$0")"/.. || exit 1
 
 outputDir="build/sync-functions"
 
-mkdir -p "$outputDir"
```

Comment:
Removed this command to provide additional verification of the `make-sync-function` script's ability to automatically create the sync function output directory.